### PR TITLE
Moderation remove

### PR DESCRIPTION
**Pull request by mehmetalisavas**
Clean topic moderation and links between channels
## Description

Removing topic moderation and channel links
## Motivation and Context

Required to clean socialapi

### CODE REVIEW

File: go/src/socialapi/workers/algoliaconnector/algoliaconnector/synonyms.go [hunk 1]
```diff
@@ -1,81 +1,6 @@
 package algoliaconnector
 
-import (
-	"errors"
-	"socialapi/models"
-)
-
-func (f *Controller) ChannelLinkCreated(cl *models.ChannelLink) error {
```
Comment (cihangir) [inline go/src/socialapi/workers/algoliaconnector/algoliaconnector/synonyms.go]: i even forgot this feature 😂

**Review by cihangir — APPROVED** [review]
we can merge after the deploy

File: go/src/socialapi/workers/algoliaconnector/algoliaconnector/synonyms.go (hunk 1 above)
Comment (mehmetalisavas) [inline go/src/socialapi/workers/algoliaconnector/algoliaconnector/synonyms.go]: my first and last FE work :):)

**Comment by cihangir** [comment]
@mehmetalisavas could you please rebase this?

**Comment by mehmetalisavas** [comment]
@cihangir once https://github.com/koding/koding/pull/9352 is merged, im gonna rebase this again

**Comment by cihangir** [comment]
@mehmetalisavas other PR is merged, you can rebase this.

**Comment by mehmetalisavas** [comment]
@cihangir rebased :)

**Review by cihangir — APPROVED** [review]

**Pull request merged**